aws-api/src/main/java/com/amplifyframework/api/aws/AppSyncGraphQLOperation.java

```diff
@@ -153,9 +153,11 @@ public void onResponse(@NonNull Call call, @NonNull Response response) {
 
         @Override
         public void onFailure(@NonNull Call call, @NonNull IOException exception) {
-            onFailure.accept(new ApiException(
-                "OkHttp client request failed.", exception, "See attached exception for more details."
-            ));
+            if (!call.isCanceled()) {
+                onFailure.accept(new ApiException(
+                        "OkHttp client request failed.", exception, "See attached exception for more details."
+                ));
+            }
```

Comment:
what is the expected behavior if the call is canceled? Will there be an appropriate callback?

Reply:
The callback after the canceled call leads to Crash - ( UndeliverableException: The exception could not be delivered to the consumer because it has already canceled/disposed the flow )

Reply:
discussed offline that canceled event is already handled at higher level